src/server-entry.js

```diff
@@ -15,8 +14,13 @@ const isDev = process.env.NODE_ENV !== 'production';
 // Since data fetching is async, this function is expected to
 // return a Promise that resolves to the app instance.
 export default context => {
-	return new Promise((resolve, reject) => {
-		const s = isDev && Date.now();
+	const s = isDev && Date.now();
+
+	// Set the webpack public path at runtime
+	global.publicPath = context.config.publicPath;
```

Comment:
@emuvente did you find issues with just adding "__webpack_public_path__ = config.publicPath || '/';" here?

It seemed to do the trick for me without changing the location of the import to preceed the new Promise...

Reply:
I guess I assumed that wouldn't work and didn't even try. That's much simpler than what I was attempting